usage/rest-api/src/main/java/org/apache/brooklyn/rest/domain/CatalogItemSummary.java

```diff
@@ -140,7 +166,7 @@ public String toString() {
 
     @Override
     public int hashCode() {
-        return Objects.hashCode(symbolicName, version, name, javaType, deprecated);
+        return Objects.hashCode(symbolicName, version, name, javaType, tags, deprecated);
```

Comment:
Ah, I see where EqualsBuilder came from. I guess when a class uses `EqualsBuilder` we should also use `HashCodeBuilder`. (Or, in my preference, not use `EqualsBuilder`!) This class disobeys the equals/hashcode contract. Not your fault, of course.